src/neural-network.js

```diff
@@ -223,13 +223,36 @@ export default class NeuralNetwork {
     return output;
   }
 
+  /**
+   *
+   * @param data
+   * @returns sizes
+   */
+  _getSizesFromData (data) {
+    let sizes = [];
+    let inputSize = data[0].input.length;
+    let outputSize = data[0].output.length;
+    let hiddenSizes = this.hiddenSizes;
+    if (!hiddenSizes) {
+      sizes.push(Math.max(3, Math.floor(inputSize / 2)));
```

Comment:
If someone doesn't want hidden layers, the library will force it?

Reply:
yep, that's how it is currently setup.

Reply:
Why? Can @robertleeplummerjr she's some light on this?

Reply:
Sorry, I read this as "yeah I did it this way, what of it?" Not as you probably intended it, "maintaining current functionality"

Reply:
haha no worries, it's finding the balance between not changing too much but still changing things in a good way :)

Reply:
Thats how it was when we got here, however one could do: `{ hiddenSizes: [] }`